user/languages/objective-c.md

```diff
@@ -41,8 +41,8 @@ the following table:
 
 ## Default Test Script
 
-Travis CI runs [xctool](https://github.com/facebook/xctool) by default to
-execute your tests. In order for xctool to work, you need to tell it where to
+Travis CI runs xcodebuild and [xcpretty](https://github.com/supermarin/xcpretty) by default to
+execute your tests. In order for xcodebuild to work, you need to tell it where to
 find your project or workspace and what scheme you would like to build. For
```

Comment:
I believe now you need to tell it where to find your project or workspace, what scheme you'd like to build and the device or simulator they'd like to use. So maybe this can change to: 

> In order for xcodebuild to work, you need to tell it where to find your project or workspace, what scheme you would like to build and which device or simulator you would like to use.

Thoughts?

Reply:
That sounds good to me!